django/core/management/commands/migrate.py

```diff
@@ -99,7 +93,7 @@ def add_arguments(self, parser):
     def handle(self, *args, **options):
         database = options["database"]
         if not options["skip_checks"]:
-            self.check(databases=[database])
+            self.check(tags=self.requires_system_checks, databases=[database])
```

Comment:
I think we might need to make further changes here as in cases where `requires_system_checks` is overridden `BaseCommand.execute` will also call `self.check` but without `databases` result in the following calls

1. `self.checks(tags=self.requires_system_checks)`
2. `self.checks(tags=self.requires_system_checks, databases=[database])`

https://github.com/django/django/blob/857b1048d53ebf5fc5581c110e85c212b81ca83a/django/core/management/base.py#L452-L459